Layout revisions

alexandercyu opened the pull request:
Changes layout so it is better for mobile viewing.

- The Scientific Experts and Learn More sections are on top of each other instead of side by side

alexandercyu committed:
Update daccs-project-css.css

Added css to change colour of github link

alexandercyu committed:
Update index.html

Changed table to use jinja function

alexandercyu committed:
Update index.html

Changed layout so it will be better for mobile viewing

mishaschwartz commented on static/css/daccs-project-css.css:
```diff
@@ -10,4 +10,9 @@ body{
     width: 25px;
+}
+
+.marble-github-link{
```
let's use "daccs" instead of "marble" here

alexandercyu replied:
Changed class to use "daccs"

mishaschwartz commented on templates/site/index.html:
```diff
@@ -22,80 +22,63 @@ <h1 class="text-center">{{ page_heading }}</h1>
+{% set table_data = [
+                {"name": "Steve Easterbrook", "institution": "University of Toronto", "department": "Computer Science"},
+                {"name": "Samuel Foucher", "institution": "Centre de recherche informatique de Montréal (CRIM)", "department": "Vision and Imaging Team"},
+                {"name": "Nathan Gillett", "institution": "Climate Research Division", "department": "Environment and Climate Change Canada"},
```
as discussed: swap institution and department

alexandercyu replied:
Swapped institution and department for Nathan Gillett.

mishaschwartz commented on templates/site/index.html:
```diff
@@ -22,80 +22,63 @@ <h1 class="text-center">{{ page_heading }}</h1>
+                {"name": "Steve Easterbrook", "institution": "University of Toronto", "department": "Computer Science"},
+                {"name": "Samuel Foucher", "institution": "Centre de recherche informatique de Montréal (CRIM)", "department": "Vision and Imaging Team"},
+                {"name": "Nathan Gillett", "institution": "Climate Research Division", "department": "Environment and Climate Change Canada"},
+                {"name": "David Huard", "institution": "Ouranos", "department": "Scénarios et services climatiques"},
```
as discussed: change to english translation

mishaschwartz commented on templates/site/index.html:
```diff
@@ -22,80 +22,63 @@ <h1 class="text-center">{{ page_heading }}</h1>
+            <tr>
+                <th>Name</th>
+                <th>Institution</th>
+                <th>Department</th>
```
as discussed: change to "Expertise"

alexandercyu replied:
Changed to "Expertise"

alexandercyu committed:
Update daccs-project-css.css

Changed name to "daccs-github-link"

alexandercyu committed:
Update index.html

- Swapped "Institution" and "Department" for Natan Gillett
- Changed "Institution" to "Affiliation" as discussed in meeting
- Changed "Department" to "Expertise" as discussed in meeting

alexandercyu committed:
Update index.html

Changed French names to English

alexandercyu committed:
Update index.html

- Added "Machine Learning" to Steve Easterbrook's expertise
- Changed "Exec" to "Executive"
- Removed comma at end of table_data array

mishaschwartz reviewed (approved):
Looks good. Are you planning on doing the other content changes (to the main paragraph) as part of this PR or in a different one?
